### PR TITLE
Add tv app

### PR DESCRIPTION
**Pull request by louiszawadzki**
### What does this PR do?

Add a sample AndroidTV and tvOS app.

### CODE REVIEW

File: rum-react-native-tvos/android/app/debug.keystore
Comment (0xnm) [inline rum-react-native-tvos/android/app/debug.keystore:1]: needs to be removed

File: rum-react-native-tvos/.buckconfig [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
```
[the remark this answers is not in the document]
Comment (louiszawadzki) [reply rum-react-native-tvos/.buckconfig:1]: I believe this hasn't been removed in the tvOS app template, will remove it

File: rum-react-native-tvos/android/app/build_defs.bzl
Comment (0xnm) [inline rum-react-native-tvos/android/app/build_defs.bzl:1]: this file is not needed as well.

**Review by 0xnm — APPROVED** [review]

File: rum-react-native-tvos/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# React Native tvOS and AndroidTV apps
+
+This app is a demo React Native app that runs on tvOS and AndroidTV.
+The application is instrumented with Datadog.
+
+See the [App.tsx](./App.tsx) on how to instrument Datadog on the javascript side.
```
Comment (kayayarai) [inline rum-react-native-tvos/README.md:6]:
```suggestion
See the [App.tsx](./App.tsx) for information about how to instrument Datadog on the JavaScript side.
```

**Review by kayayarai — APPROVED** [review]
Tiny edit suggestions, but 👍

**Pull request merged**